api_tests/nodes/views/test_node_detail.py

```diff
@@ -480,8 +480,12 @@ def test_node_shows_related_count_for_linked_by_relationships(self, app, user, p
         assert res.json['data']['relationships']['linked_by_nodes']['links']['related']['meta']['count'] == 1
         assert res.json['data']['relationships']['linked_by_registrations']['links']['related']['meta']['count'] == 1
 
+        log_date = timezone.now()
+        project_private.deleted_date = log_date
+        project_private.deleted = log_date
         project_private.is_deleted = True
```

Comment:
Note: somehow, when I tried `project_private.remove_node(auth=Auth(user=user))`, the node is not deleted though the method went through the process and returned `True`.